packages/rum-core/src/domain/view/viewMetrics/trackLoadingTime.spec.ts

```diff
@@ -127,14 +127,19 @@ describe('trackLoadingTime', () => {
     clock.tick(AFTER_PAGE_ACTIVITY_END_DELAY)
 
     expect(loadingTimeCallback).toHaveBeenCalledOnceWith(
-      clock.relative(BEFORE_PAGE_ACTIVITY_VALIDATION_DELAY + CLOCK_GAP)
+      clock.relative(BEFORE_PAGE_ACTIVITY_VALIDATION_DELAY + CLOCK_GAP),
+      false
     )
   })
 
   it('should discard loading time if page is hidden before activity', () => {
     setPageVisibility('hidden')
     startLoadingTimeTracking()
 
-    expect(loadingTimeCallback).not.toHaveBeenCalled()
+    clock.tick(BEFORE_PAGE_ACTIVITY_VALIDATION_DELAY)
+    domMutationObservable.notify()
+    clock.tick(AFTER_PAGE_ACTIVITY_END_DELAY)
```

Comment:
If there is no activity, `loadingTimeCallback` is not expected to be called anyway (this is the same test as [line 56](https://github.com/DataDog/browser-sdk/blob/3c1829843c1213b913386335db310b1784bf5741/packages/rum-core/src/domain/view/viewMetrics/trackLoadingTime.spec.ts#L56-L59) and setting the visibility or not has no effect!